scripts/lib/mkcloud-onhost.sh

```diff
@@ -24,6 +24,11 @@ function onhost_setup_portforwarding
         mosh_end=$((   $cloud_port_offset + 60010 ))
 
         mkdir -p $boot_mkcloud_d
+        f [[ ${want_ipv6} == 0 ]]; then
```

Comment:
sigh, the 'if' is missing the 'i' and took me a surprisingly long time to find this issue because of mkcloud with full debugging meant too much logging to see the syntax error error.

Reply:
Yes, very subtle that one :smile: